doc/attr.xml

```diff
@@ -1211,3 +1211,60 @@ gap> OutNeighbours(sym);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="UndirectedSpanningTree">
+<ManSection>
+  <Attr Name="UndirectedSpanningTree" Arg="digraph"/>
+  <Attr Name="UndirectedSpanningForest" Arg="digraph"/>
+  <Returns>A digraph, or <K>fail</K>.</Returns>
+  <Description>
+    If <A>digraph</A> is a digraph with at least one vertex, then
+    <C>UndirectedSpanningForest</C> returns an undirected spanning forest of
+    <A>digraph</A>, otherwise this attribute returns <K>fail</K>. See <Ref
+      Oper="IsUndirectedSpanningForest" /> for the definition of an undirected
+    spanning forest.
+
+    <P/>
+
+    If <A>digraph</A> is a digraph with at least one vertex and whose <Ref
+      Attr="MaximalSymmetricSubdigraph"/> is connected (see <Ref
+      Prop="IsConnectedDigraph" />), then <C>UndirectedSpanningTree</C> returns
+    an undirected spanning tree of <A>digraph</A>, otherwise this attribute
+    returns <K>fail</K>.  See <Ref Oper="IsUndirectedSpanningTree" /> for the
+    definition of an undirected spanning tree.
+
+    <P/>
+
+    Note that for a digraph that has an undirected spanning tree, the attribute
+    <C>UndirectedSpanningTree</C> returns the same digraph as does the attribute
+    <C>UndirectedSpanningForest</C>.
+
+    <P/>
+
+    <Example><![CDATA[
+gap> gr := Digraph([[1, 2, 1, 3], [1], [4], [3, 4, 3]]);
+<multidigraph with 4 vertices, 9 edges>
+gap> forest := UndirectedSpanningForest(gr);;
+gap> OutNeighbours(forest);
+[ [ 2 ], [ 1 ], [ 4 ], [ 3 ] ]
+gap> IsUndirectedSpanningForest(gr, forest);
+true
+gap> DigraphConnectedComponents(forest).comps;
+[ [ 1, 2 ], [ 3, 4 ] ]
+gap> DigraphConnectedComponents(MaximalSymmetricSubdigraph(gr)).comps;
+[ [ 1, 2 ], [ 3, 4 ] ]
+gap> UndirectedSpanningTree(gr);
+fail
```

Comment:
The purpose of this line is to show that a digraph does not have a spanning tree if its underlying symmetric "graph" is not connected. We've already calculated variable `forest := UndirectedSpanningForest(gr)` earlier in the example. I'll add something similar.